setuptools/config/_apply_pyprojecttoml.py

```diff
@@ -33,11 +33,12 @@
     from distutils.dist import _OptionsList
     from setuptools._importlib import metadata
     from setuptools.dist import Distribution
+    from typing_extensions import TypeAlias
 
-EMPTY: Mapping = MappingProxyType({})  # Immutable dict-like
-_ProjectReadmeValue = Union[str, Dict[str, str]]
-_CorrespFn = Callable[["Distribution", Any, StrPath], None]
-_Correspondence = Union[str, _CorrespFn]
+EMPTY: MappingProxyType = MappingProxyType({})  # Immutable dict-like
```

Comment:
I think it makes more sense to preserve `Mapping` here... If Python had a immutable dict-object I would use that... `MappingProxyType` is just a implementation detail.

Reply:
```suggestion
EMPTY: Mapping = MappingProxyType({})  # Immutable dict-like
```